### PR TITLE
Update TileSet Builder version to Godot 3.2

### PR DESCRIPTION
**Pull request by rich1richard**
updates line 171: removes *(Godot 2.x)*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -168,7 +168,7 @@ A curated list of [free/libre](https://www.gnu.org/philosophy/free-sw.html) game
 - [SUTjoystick](https://gitlab.com/shine-upon-thee/joystick) - Easy gamepad support for GNU/Linux and Windows (Godot 2.1).
 - [Tiled importer](https://github.com/vnen/godot-tiled-importer) - Import maps from [Tiled](https://www.mapeditor.org/) (Godot 3.1).
 - [Tileset Builder](https://gist.github.com/Calinou/27e979ab0a35500c3381) - Build tilesets.
-- [TileSet Builder](https://github.com/HeavenMercy/TileSet-Builder-Godot-Plugin) - Quickly build tilesets with style (Godot 2.x).
+- [TileSet Builder](https://github.com/HeavenMercy/TileSet-Builder-Godot-Plugin) - Quickly build tilesets with style.
```
Comment (Calinou) [inline README.md]: The Godot version should always be mentioned:

```suggestion
- [TileSet Builder](https://github.com/HeavenMercy/TileSet-Builder-Godot-Plugin) - Quickly build tilesets with style (Godot 3.2).
```

**Review by Calinou — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rich1richard) [reply README.md]: I have a branch of the old Godot 2 version (godot_2)
I just added a version  for Godot 3 and made it as my default branch (master)

I've seen and heard that some are still using Godot 2.x
I don't know... Is it really necessary to specify a version ?

Comment (rich1richard) [reply README.md]: I've added a line for **Godot 2.x** in the master's **README.md**

Comment (Calinou) [reply README.md]: > I don't know... Is it really necessary to specify a version ?

Yes. This is to make it easier for people to find add-ons that are compatible with the Godot version they're using :slightly_smiling_face: 

The list still mentions a lot of a Godot 2.x add-ons, but they're not relevant to most people nowadays. (I plan to remove them in the future.)

Comment (rich1richard) [reply README.md]: Done! 😄

**Review by Calinou — APPROVED** [review]
Thanks!

**Pull request merged**